docs/PLUGIN_AUTHORING.md

```diff
@@ -127,7 +127,7 @@ In the plugin repo root directory, the `metadata.yml` file provides Snap project
 * **description**: paragraph describing the plugin's purpose
 * **badge**: a list of [badges](https://shields.io/) to display
 * **ci**: a list of ci services running for this repo
-* **status**: one of the three statuses [described below](#plugin-status)
+* **status**: one of the four statuses [described below](#plugin-status)
```

Comment:
This implies that the four statuses are described below. When in reality the four statuses are described in Plugin_Status.md